play-services-fido/src/main/aidl/com/google/android/gms/fido/fido2/internal/regular/IFido2RegularService.aidl

```diff
@@ -0,0 +1,14 @@
+package com.google.android.gms.fido.fido2.internal.regular;
+
+import com.google.android.gms.fido.fido2.internal.regular.IFido2RegularCallbacks;
+import com.google.android.gms.fido.fido2.api.IBooleanCallback;
+import com.google.android.gms.fido.fido2.api.ICredentialListCallback;
+import com.google.android.gms.fido.fido2.api.common.PublicKeyCredentialCreationOptions;
+import com.google.android.gms.fido.fido2.api.common.PublicKeyCredentialRequestOptions;
+
+interface IFido2RegularService {
```

Comment:
I'm a little confused about these interfaces, because they don't match what Google's services expect:
- The name should be `IFido2AppService` (`IFido2AppCallbacks` respectively)
- The AIDL shouldn't carry a `getCredentialList` function on the regular service (such function also doesn't exist in the corresponding [public API](https://developers.google.com/android/reference/com/google/android/gms/fido/fido2/Fido2ApiClient?hl=en))

Not sure where this mismatch comes from. If you only ever tested the client library from this PR against the service from this PR, it would certainly work, but would be incompatible with Google's client library or service.

Reply:
That may be why. I have converted to a draft until this is tested against a google client lib, and this fixed.